deploy/kubernetes_operator/pkg/operator/event_handler.go

```diff
@@ -33,6 +34,8 @@ import (
 	pb "github.com/bytedance/fedlearner/deploy/kubernetes_operator/proto"
 )
 
+const clientTimeout = time.Second * 1
```

Comment:
May be one second is little short for public network connection some times. 3-5 seconds may be better.

Reply:
Good point, fixed